### PR TITLE
Adding task 110

### PR DESCRIPTION
**Pull request by Mihir3009**

### CODE REVIEW

**Comment by Mihir3009** [comment]
@swarooprm please merge this. Thanks.

**Comment by danyaljj** [comment]
Let's the definition more crips (e.g.,`in text form`: why is this needed?)

Here is a rephrasing: `In this task, you are given commands (in terms of logical operations) to select relevant rows from the given table. Your job is to generate a natural language interpretation of the given command.`


More importantly, we need to define the syntax of the logical operation in the input definition. Basically, let's briefly define what all the recurring keywords/operations (`eq`,  `hop` `argmax`, `all_rows`, etc.) mean.

**Comment by danyaljj** [comment]
pinging the thread to make sure you have seen the comments above.

**Comment by swarooprm** [comment]
@Mihir3009 
Can you only commit the task110 file and readme file? Currently, it shows 15 changed files.

**Comment by Mihir3009** [comment]
Yes

**Comment by Mihir3009** [comment]
I am having trouble in resolving the issue.

**Comment by Mihir3009** [comment]
@danyaljj @swarooprm Do I need to define all the recurring keywords/operations (eq, hop argmax, all_rows, etc.) in the definition or in the input examples? There are 22 of them.

**Comment by Mihir3009** [comment]
please review this updated file.

File: tasks/README.md [hunk 1]
```diff
@@ -103,8 +103,11 @@ Name | Summary | Category
 `task098_conala_list_intersection` | Given a two lists of numbers find the intersection of the two lists | Answer Generation, Arithmetic
 `task_111_asset_sentence_simplification` | Given a sentence, simplify it so it can be understood by non-native English speakers | Generation, Paraphrasing
 `task_112_asset_simple_sentence_identification` | Given two text pieces, choose the one that is simpler and easier to understand by non-native speakers | Answer Generation, Sentence Comparison
+`task102_commongen_sentence_generation` | Given a collection of concepts, use them in a coherent sentence. | Sentence Generation 
+`task103_facts2story_long_text_generation` | Given 5 facts, write a story that incorporates them. | Long Text Generation 
 `task102_commongen_sentence_generation` | Given a collection of concepts, use them in a coherent sentence. | Sentence Generation
 `task103_facts2story_long_text_generation` | Given 5 facts, write a story that incorporates them. | Long Text Generation
+>>>>>>> b09e18ee9ff8e5dbc62e67c5275f62927da0be44
```
Comment (danyaljj) [inline tasks/README.md:110]: drop this line

File: tasks/README.md [hunk 2]
```diff
@@ -103,8 +103,11 @@ Name | Summary | Category
 `task098_conala_list_intersection` | Given a two lists of numbers find the intersection of the two lists | Answer Generation, Arithmetic
 `task_111_asset_sentence_simplification` | Given a sentence, simplify it so it can be understood by non-native English speakers | Generation, Paraphrasing
 `task_112_asset_simple_sentence_identification` | Given two text pieces, choose the one that is simpler and easier to understand by non-native speakers | Answer Generation, Sentence Comparison
+`task102_commongen_sentence_generation` | Given a collection of concepts, use them in a coherent sentence. | Sentence Generation 
+`task103_facts2story_long_text_generation` | Given 5 facts, write a story that incorporates them. | Long Text Generation 
 `task102_commongen_sentence_generation` | Given a collection of concepts, use them in a coherent sentence. | Sentence Generation
 `task103_facts2story_long_text_generation` | Given 5 facts, write a story that incorporates them. | Long Text Generation
```
Comment (danyaljj) [inline tasks/README.md:109]: the merge is not done right here. You have repeated lines here.

File: tasks/README.md [hunk 3]
```diff
@@ -113,6 +116,10 @@ Name | Summary | Category
 `task113_count_frequency_of_letter.json` | Count Frequency of a letter in the given string | Answer Generation
 `task114_is_the_given_word_longest.json` | Is the given word longest in the sentence | Classification
 `task115_Help_advice_classification` | Given a text detect whether it's an advise or not. | Classification
+`task117_spl_translation_en_de.json` | Translate English questions to German | Translation
+`task113_count_frequency_of_letter.json` | Count Frequency of a letter in the given string | Answer Generation
+`task114_is_the_given_word_longest.json` | Is the given word longest in the sentence | Classification
+`task115_Help_advice_classification` | Given a text detect whether it's an advise or not. | Classification
```
Comment (danyaljj) [inline tasks/README.md:122]: Again, repeated lines because of incorrect merge.

**Comment by danyaljj** [comment]
Thanks for updating the task definition. Here are several suggestions: 

 - `logic forms` -> `logical operators` 
 - `used in this dataset` -> drop 
 - Several issues in the task readme file. 

Other than these, I am happy with this PR.

**Comment by Mihir3009** [comment]
Because of unexpected errors with this branch:

remote: Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.
remote: Please see https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information.
fatal: unable to access 'https://github.com/Mihir3009/natural-instructions-expansion.git/': The requested URL returned error: 403

I created another pull request with all changes suggested in task110. Please merge that pull request.

**Pull request closed without merging**